transmart-rest-api/src/main/groovy/org/transmartproject/rest/serialization/HypercubeJsonSerializer.groovy

```diff
@@ -164,6 +166,9 @@ class HypercubeJsonSerializer extends HypercubeSerializer {
             writer.nullValue()
         } else if (value instanceof String) {
             writer.value((String) value)
+        } else if (value instanceof Timestamp) {
+            def time = ((Timestamp)value).toLocalDateTime().atZone(ZoneId.of("GMT")).toInstant().toString()
```

Comment:
@baroleg Thanks for contributing. Would it be possible for you to write a simple regression test that reproduces the issue?
See `transmart-core/transmart-rest-api/src/test/groovy/org/transmartproject/rest/serialization/` for test examples.